magichome/magichome.js

```diff
@@ -268,7 +245,7 @@ module.exports = function (RED) {
         }
 
         function sendStateMsg() {
-            var msg = {payload:internalState.status};
+            const msg = { payload: internalState.status };
```

Comment:
why const and not let?

Reply:
I thought const would fit better, because msg should not be reassigned in this scope.